static/styles/portico/portico_signin.css

```diff
@@ -95,6 +95,20 @@ html {
 
             margin: 0 20px;
         }
+
+        select {
+            height: 30px;
+            padding: 4px 6px;
+            width: 220px;
+            font-size: 14px;
+            color: hsl(0, 0%, 33%);
+            border-radius: 4px;
+            border: 1px solid hsl(0, 0%, 80%);
+            margin-bottom: 10px;
+            cursor: pointer;
+            background-color: hsl(0, 0%, 100%);
+            font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
```

Comment:
Have added `font-family` here since rest of the page uses the same font, and it would be a big change to change the font of the page itself.